FurEver/wwwroot/css/quiz.css

```diff
@@ -0,0 +1,82 @@
+﻿﻿ /* quiz.css */
+.container {
+    max-width: 600px;
+    margin: auto;
+    text-align: center;
+    font-family: Arial, sans-serif;
+}
+
+h1 {
+    font-size: 2em;
+    margin-bottom: 20px;
+    color: #333;
+}
+
+.dog-image {
+    margin-bottom: 20px;
+}
+
+    .dog-image img {
+        width: 50%;
```

Comment:
# Technical Change #1
## Shrinking down the size of the images of a dog breed when trying to complete the breed identification quiz
Implementing this change reduces technical debt by increasing both accessibility and usability, while complying with web standards. It specifically aligns with WCAG 2.1 guideline 1.4.10 Reflow. This guideline ensures users can interact with content without excessive scrolling or keyboard navigation. By addressing the image sizes of the api proactively, the code becomes more compliant and reduces the likelihood of future rework to meet accessibility requirements. 
Link to Reference: https://www.w3.org/WAI/WCAG22/quickref/?versions=2.1&showtechniques=131%2C133%2C134%2C148%2C1410%2C243%2C321%2C412&currentsidebar=%23col_customize&technologies=html%2Cwai-aria%2Cjs%2Cserver%2Csmil%2Cpdf&hidesidebar=true#focus-visible

Reply:
This is good, though I'd go one step further and resize the original image.
If you have a large image that you resize on the client (via HTML), you're downloading more than you need to, which means:
- you're eating up bandwidth
- the page load time is slower.